### PR TITLE
Update Cinnie

### PR DESCRIPTION
**Pull request by yuu1031**
add sth.

### CODE REVIEW

**Review by Jerrylum — CHANGES_REQUESTED** [review]
1.html is not a name

File: 2.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>
+        hello! I'm Cinnie~!!
```
Comment (Jerrylum) [inline 2.html:10]: Remove `~` symbol

File: 2.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>
+        hello! I'm Cinnie~!!
+    </h1>
+    <h3>hey~~</h3>
```
Comment (Jerrylum) [inline 2.html:12]: Remove

**Review by Jerrylum — CHANGES_REQUESTED** [review]
Change 2 items please

File: 2.html (hunk 2 above)
Comment (yuu1031) [reply 2.html:12]: no

Comment (Jerrylum) [reply 2.html:12]: okay fine

**Pull request merged**